book/src/configuration.md

```diff
@@ -45,6 +45,8 @@ hidden = false
 | `auto-info` | Whether to display infoboxes | `true` |
 | `true-color` | Set to `true` to override automatic detection of terminal truecolor support in the event of a false negative. | `false` |
 | `rulers` | List of column positions at which to display the rulers. Can be overridden by language specific `rulers` in `languages.toml` file. | `[]` |
+| `spinner` | List of strings comprising the frames to be used for the progress spinner[^1]. | `["⣾","⣽","⣻","⢿","⡿","⣟","⣯","⣷"]` |
```

Comment:
Let's call the setting `spinner-frames` instead. I think that matches the vector value better than the singular "spinner"

Reply:
Done!